controllers/selfnoderemediationconfig_controller.go

```diff
@@ -160,7 +160,9 @@ func (r *SelfNodeRemediationConfigReconciler) syncConfigDaemonSet(ctx context.Co
 		}
 		err = r.syncK8sResource(ctx, snrConfig, obj)
 		if err != nil {
-			logger.Error(err, "Couldn't sync self-node-remediation daemons objects")
+			if !errors.IsConflict(err) {
```

Comment:
IMHO hiding the error leads to incomplete logs. We will see a requeue and don't know the reason for it 🤔 What about logging it as info at least?